### PR TITLE
Update online labeling crop generator. 

### PR DESCRIPTION
**Pull request by Arkkienkeli**
Make online_labels_cropgen.py consistent with the latest changes.
The `LAMBDA` and `online_label_smoothing` are parameterized in the config.
Closes https://github.com/cytomining/DeepProfiler/issues/290

### CODE REVIEW

**Review by jccaicedo — CHANGES_REQUESTED** [review]
Remove distinction between training and validation for randomizing data (around line 75).
Unify the LABEL_SMOOTHING parameter with the parameter in the Keras loss function (automatically set to zero when using this crop generator).

**Comment by jccaicedo** [comment]
Thanks for the PR, @Arkkienkeli !
